### PR TITLE
Feat/display

### PR DESCRIPTION
**Pull request by CapCinematic**
- files changed: app.tsx, singlCard.tsx, CardContainer
- change summary: We went through the process of understanding Props and Interface through TypeScript, now we have a clearer understanding of how this data should be passed we have data being displayed on the dom.
- next up: creating cards and links to brands.

### CODE REVIEW

**Review by brendel-r — APPROVED** [review]
everything looks great! we finally made it through some typescript!

File: src/components/App/App.tsx [hunk 1]
```diff
@@ -18,11 +27,20 @@ export class App extends Component {
     }
   }
 
+  componentDidMount(){
+    acquireInfo()
+    .then(data => {
+      console.log(data)
+      this.setState({ allMakeUp: data })
+    })
```
Comment (niksseif) [inline src/components/App/App.tsx:35]: Make sure to add a catch block here and set the error state within the catch block.

File: src/components/App/App.tsx [hunk 2]
```diff
@@ -18,11 +27,20 @@ export class App extends Component {
     }
   }
 
+  componentDidMount(){
+    acquireInfo()
+    .then(data => {
+      console.log(data)
```
Comment (niksseif) [inline src/components/App/App.tsx:33]: Make sure to remove console.logs before pushing code up.

File: src/Assets/BrandNames.ts [hunk 3]
```diff
@@ -0,0 +1,60 @@
+
+const brand_array = [
```
Comment (niksseif) [inline src/Assets/BrandNames.ts:2]: What happens if this API adds a new brand to their database? It's OK for MVP but later think about how would you change this to a more dynamic approach.

File: src/utilities.tsx [hunk 4]
```diff
@@ -3,24 +3,24 @@ export interface IProduct {
   brand: string,
   name: string,
   price: string,
-  price_sign: null,
-  currency: null,
+  // price_sign: null,
```
Comment (niksseif) [inline src/utilities.tsx:6]: Make sure to remove dead-code/commented out code before pushing code to github.

**Review by niksseif — COMMENTED** [review]
Nice work everyone.

**Pull request merged**